### PR TITLE
Saved plant list

### PR DESCRIPTION
**Pull request by MrymSafavi**
saved plant list in plant app updated

### CODE REVIEW

File: accounts/admin.py [hunk 1]
```diff
@@ -19,3 +19,4 @@ class NormalUserAdmin(admin.ModelAdmin):
 
 
 admin.site.unregister(Group)
+# admin.site.register(PlantUserRelationship)
```
Comment (mohamad-khorsandi) [inline accounts/admin.py]: delete

File: accounts/models.py [hunk 2]
```diff
@@ -40,6 +43,12 @@ def __str__(self):
         return self.email
 
 
+# class PlantUserRelationship(models.Model):
```
Comment (mohamad-khorsandi) [inline accounts/models.py]: remove this

**Review by mohamad-khorsandi — CHANGES_REQUESTED** [review]

File: accounts/admin.py (hunk 1 above)
Comment (MrymSafavi) [reply accounts/admin.py]: ok

File: accounts/models.py (hunk 2 above)
Comment (MrymSafavi) [reply accounts/models.py]: ok.

**Review by mohamad-khorsandi — APPROVED** [review]

**Pull request merged**